gsa/src/gmp/models/__tests__/scanconfig.js

```diff
@@ -119,7 +119,7 @@ describe('ScanConfig model tests', () => {
     const scanConfig = new ScanConfig(elem);
 
     expect(scanConfig.families.count).toEqual(42);
-    expect(scanConfig.families.trend).toEqual('1');
+    expect(scanConfig.families.trend).toEqual(1);
```

Comment:
the new constant should be used for the test here too
```suggestion
    expect(scanConfig.families.trend).toEqual(SCANCONFIG_TREND_DYNAMIC);
```